### DISCUSSION TITLE
My private contributions have not been counted in the card

### DISCUSSION THREAD

**Discussion by walifile** [anuraghazra/github-readme-stats#3676] [Q&A] [answered]
I'm having trouble displaying my private commits on my GitHub README despite deploying my own instance and granting all permissions when creating a token. Has anyone encountered a similar issue or found a solution? I've double-checked my settings, but still can't seem to get it to work. Any insights or suggestions would be greatly appreciated!

**Comment by qwerty541** [comment]
Hey, @walifile! Make sure that you enabled `repo` and `user` permissions for your personal access token.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/9c94bb06-a59d-402e-8014-5f821046b630)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c835b969-9850-4cc9-82ed-b9207214b47f)

**Reply by walifile** [reply]
Hi @qwerty541 I already did this still getting the same issue. Thanks

**Reply by qwerty541** [reply]
Try to create a new personal access token with same permissions as on my screenshots and replace your current.

**Reply by walifile** [reply]
@qwerty541  I have created a new personal access token with the same permissions as shown in your screenshots but unfortunately I am still encountering the same issue. Do you have any other suggestions for troubleshooting? . Thanks

**Reply by qwerty541** [reply]
Currently README file in your GitHub profile contains two cards with links on the following deployments: `https://github-readme-stats.vercel.app` and `https://github-readme-stats-eta-green.vercel.app`. `https://github-readme-stats.vercel.app` is public deployment which does not have access to your private contributions and never will. The next one `https://github-readme-stats-eta-green.vercel.app`. as I understand. it is your deployment. You can visit the following URL https://github-readme-stats-eta-green.vercel.app/api/status/pat-info to check that it currently using expired token and doesn't work. Now you need to update `PAT_1` environment variable value on your Vercel deployment with new personal access token which you have recently created and than replace `https://github-readme-stats.vercel.app` with `https://github-readme-stats-eta-green.vercel.app` in your README.

**Reply by walifile** [reply]
@qwerty541 I have updated the access token in the environment variable (PAT_1) on my Vercel deployment. However its still showing as expired. I have double-check the token and ensure its correctly configured. Please help me on this.

For the time being I have included both URLs in my README. The one pointing to https://github-readme-stats.vercel.app/ is for public deployments, while the one pointing to https://github-readme-stats-eta-green.vercel.app/ is intended for private access. I will remove the public URL once the private deployment is functioning correctly. Thanks for your patience

**Reply by qwerty541** [reply]
If you sure that both of your personal access token and envorinment variable was created and configured correctly, then you can try to redeploy your vercel instance. Redeploy happens automatically when you pushing to the deployment branch in your forked repository, so you can just merge latest changes from upstream repository into master branch of your fork. Also you can perform redeploy using Vercel CLI https://vercel.com/docs/cli/redeploy.

In case if deployment still show that your token is expired after completing these steps, then it is something wrong with your token. You can try to create a new token without expiration date.

Also please clarify whether you are using classic or fine-grained personal access token.

**Reply by walifile** [reply, marked as answer]
@qwerty541  Success :)  I managed to resolve the issue by deleting the project from Vercel and redeploying it. If you have any questions feel free to ask. Otherwise I will be closing the discussion and  I wil create a YouTube video  on this as well. Thanks for the guidance.

**Reply by qwerty541** [reply]
I'm glad that you have resolved your issue. Please, mark this discussion as answered if you don't need further assistace.
